src/main.py

```diff
@@ -24,7 +24,7 @@ async def lifespan(app: FastAPI):
 app.include_router(fastapi_users_ep.get_reset_password_router(), prefix="/auth", tags=["auth"])
 app.include_router(fastapi_users_ep.get_verify_router(UserRead), prefix="/auth", tags=["auth"])
 app.include_router(fastapi_users_ep.get_users_router(UserRead, UserUpdate), prefix="/auth", tags=["auth"])
-app.include_router(router=mortgage_router,prefix="/scrap", tags=["scraping"])
+app.include_router(router=mortgage_router, prefix="/scrap", tags=["scraping"])
```

Comment:
nit: `/scrap` -> `/scrapper`

Reply:
changed the prefix /scrap to /scrapper in mortgage_router